.github/workflows/test.yaml

```diff
@@ -9,16 +9,19 @@ on:
       working-directory:
         type: string
         description: The working directory for jobs
-        default: './'
+        default: "./"
+      self-hosted-runner:
+        type: boolean
+        description: Whether to use self-hosted runners to run the jobs.
+        default: false
```

Comment:
Should we make the default to use the self-hosted runners? Or perhaps we do that after we scale them?

Reply:
I think it might be good to set the default to true. If developers are encountering issues when they are working a PR, they can set it `self-hosted-runner` to false.